src/components/Records/Record/GeneralInfo/Citations.vue

```diff
@@ -28,7 +28,7 @@
             </span>
             <b v-if="getField('doi')">DOI:</b>{{ getField('doi') }};
             <b v-if="getField('lastEdited')">Last Edited:</b>{{ getField('lastEdited') }}
-            <b>Last Accessed:</b>{{ getCurrentDate() }}
+            <b>Last Accessed:</b>{{ new Date() | moment("dddd, MMMM Do YYYY") }}
```

Comment:
I note that the time is not displayed here; in the old system it is, and so ought to be added for consistency.

Reply:
okay it will be added.

Reply:
![image](https://user-images.githubusercontent.com/25683981/106912428-ea147b00-66fa-11eb-8502-76a9b0ff2e3a.png)
done

Reply:
Thanks. I'd prefer to see that in 24-hour time as 15:08 (we probably don't need the seconds either), otherwise I think that should do it.

Reply:
okay I remove second and also turn It into a 24-hour time

Reply:
@knirirr 
your request is done. 24 hour time in new commit. 

![image](https://user-images.githubusercontent.com/25683981/106929143-e76e5180-670b-11eb-80e2-282e38d10e7d.png)

Reply:
We inch closer... just the "am/pm" to dispense with now.

Reply:
![image](https://user-images.githubusercontent.com/25683981/106958932-ec45fc00-6731-11eb-83d4-1c63bcc54f72.png)

done.

Reply:
Please remove the "now" then it should be ready to go.

Reply:
![image](https://user-images.githubusercontent.com/25683981/107217338-f4d65500-6a05-11eb-83a2-1b64beb50564.png)
@knirirr please check

Reply:
![image](https://user-images.githubusercontent.com/25683981/107217338-f4d65500-6a05-11eb-83a2-1b64beb50564.png)
deleted now.